indico/modules/events/clone.py

```diff
@@ -25,13 +25,19 @@ class EventLocationCloner(EventCloner):
 
     @property
     def is_available(self):
-        return self.old_event.has_location_info
+        return self._has_content(self.old_event)
 
-    def run(self, new_event, cloners, shared_data):
+    def has_conflicts(self, target_event):
+        return self._has_content(target_event)
+
+    def run(self, new_event, cloners, shared_data, event_exists=False):
         with db.session.no_autoflush:
             self._clone_location(new_event)
         db.session.flush()
 
+    def _has_content(self, event):
+        return bool(event.has_location_info)
```

Comment:
```suggestion
        return event.has_location_info
```

it's already a bool ;)